src/main/scala/encry/EncryApp.scala

```diff
@@ -24,18 +26,22 @@ import kamon.Kamon
 import kamon.influxdb.InfluxDBReporter
 import kamon.system.SystemMetrics
 import org.encryfoundation.common.utils.Algos
+
 import scala.concurrent.{Await, ExecutionContextExecutor}
 import scala.concurrent.duration._
 import scala.io.Source
 import scala.language.postfixOps
+import net.ceedubs.ficus.Ficus._
 
 object EncryApp extends App with StrictLogging {
 
   implicit val system: ActorSystem = ActorSystem()
   implicit val ec: ExecutionContextExecutor = system.dispatcher
   implicit val materializer: ActorMaterializer = ActorMaterializer()
 
-  lazy val settings: EncryAppSettings = EncryAppSettings.read(args.headOption)
+  val cmdArgs:Array[String] = args
+
+  private lazy val settings: EncryAppSettings = EncryAppSettings.settings
```

Comment:
what the reason to make it private?

Reply:
The only point to get the settings is EncryAppSettings.settings